src/store/store.js

```diff
@@ -0,0 +1,65 @@
+import {makeAutoObservable} from 'mobx';
+class Store {
```

Comment:
1. put empty line above this line
2. in mobx you don't create 1 store for everything. It's against SOLID's first principle S - single responsibility. You create separate stores for different features